### ISSUE TITLE
Improve contrast of "read full contitions" link

### ISSUE THREAD

**Issue by hackartisan** [pulibrary/pulfalight#1776]
refs #1717 
[full list of issues for audit1717](https://github.com/pulibrary/pulfalight/issues?q=is%3Aissue%20state%3Aopen%20audit1717)

### Location of problem on site

component page with access restrictions https://findingaids.princeton.edu/catalog/C0108_c3741

### Description of error / warning
low contrast (4.33:1); contrast should be 4.5:1

### WCAG success criterion
[1.4.3 Contrast (Minimum) (Level AA)](https://webaim.org/standards/wcag/checklist#sc1.4.3)

### WCAG conformance level
AA

### Recommended fix
Probably lighten the background yellow color

### Tool used to find the problem
wave

### Screenshot

<img width="885" height="221" alt="Image" src="https://github.com/user-attachments/assets/d9b006aa-a70e-4be4-bd4e-a0e722893c5d" />
